clients/privacy-center/common/toast-options.ts

```diff
@@ -0,0 +1,21 @@
+import {UseToastOptions} from "@chakra-ui/react";
+
+const BaseToastOptions: UseToastOptions = {
+    position: "top",
+    duration: 5500
+}
```

Comment:
FYI there's this helper file in admin-ui which I plan to pull into fidesui soon which does similar stuff as this one https://github.com/ethyca/fides/blob/main/clients/admin-ui/src/features/common/toast.tsx

I do think it's nice the way you have the actual error in this file though, should make it easy to import and stay consistent with our copy! I'll have to noodle on it some more, but I might prefer the way you have here with just exporting the ToastOptions instead of what is in admin-ui of exporting a function 👍